_commands/tools/nslookup.md

```diff
@@ -0,0 +1,75 @@
+---
+---
+
+nslookup
+-------
+
+nslookup which stands for "name server lookup" is used to query internet name servers interactively for information. 
+
+~~~ bash
+nslookup [-option ...] [host-to-find | -[server]]
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+#### Example command
+~~~ 
+nslookup microsoft.com
+~~~
+##### Break it down
+The response would be: 
+
+Server:		141.211.125.17
+Address:	141.211.125.17#53
+
+Non-authoritative answer:
+Name:	microsoft.com
+Address: 23.100.122.175
+Name:	microsoft.com
+Address: 23.96.52.53
+Name:	microsoft.com
+Address: 191.239.213.197
+Name:	microsoft.com
+Address: 104.40.211.35
+Name:	microsoft.com
+Address: 104.43.195.251
+
+*Here 141.211.125.17 is the address of our system's DNS. #53 indicates that we are communicating with it on port 53, 
+which is the standard port number DNS's use to accept queries. 
+*Below this is the lookup info of microsoft.com. Here we get 5 different addresses which indicates that when you access 
```

Comment:
These aren't showing up as bullet points. I think you need to put a space between the asterisk and content.